frontend/src/components/core/Block/Block.tsx

```diff
@@ -152,11 +164,24 @@ class Block extends PureComponent<Props> {
   }
 
   private renderElementWithErrorBoundary(
-    element: SimpleElement,
+    elementWrapper: ElementWrapper,
     index: number,
     width: number
   ): ReactNode | null {
-    const component = this.renderElement(element, index, width)
+    const element = elementWrapper.get("element")
+      ? (elementWrapper.get("element") as SimpleElement)
+      : null
+
+    if (!element) {
+      return null
```

Comment:
I don't think `element` can ever be `null`. So if you want to do this check here, I think the correct reaction when the check fails would be to throw an error.

Reply:
Also, the logic above is overly complex. Why not just do:

```ts
private renderElementWithErrorBoundary( ... ) {
  if (!elementWrapper.has("element")) {
    throw new Error("Element wrapper has no element field")
  }

  const element = elementWrapper.get("element") as SimpleElement
```

Reply:
So is this still needed?